### ISSUE TITLE
(req) NPC nickname, invis armor, enchated equips

### ISSUE THREAD

**Issue by gummby8** [CitizensDev/Citizens#543]
1) I know Citizens will NEVER support custom skins, but because it does support playernames and those skins, would it be possible to add a feature to just nickname an NPC. This way you can have good looking NPCs without "john13445123killerspam" for a name

2) Would it be possible to have a toggle setting to make NPC equipped armor invisible. Thus you can actual make well armored NPCs that are still fashionable.

3) Can citizens support equipping your NPCs with enchanted armor/weapons?

**Comment by zupalex** [comment]
1) I suggested this last week so I totally agree with that.

3) It will also be cool to be able to make trader sell enchanted weapons, because right now, if you put an enchanted weapon in a trader inventeory, it appears enchanted in its inventory but loose its enchantement once you buy it.

**Comment by Loserman778** [comment]
I think that all of these suggestions should be implemented into citizens. Totally agree with all of them

**Comment by IDragonfire** [comment]
Invis armor is an interesting idea ... 

@1: You want to set the name to "Notch" (because Notch has a Skin) but the display name to "MyName", is that right?

**Comment by l1amm** [comment]
I agree with these suggestions, especially #1, would be awesome and seems like it would be easily possible!

**Comment by zupalex** [comment]
@IDragonfire : about the suggestion 1, that's exactly how I saw this.

**Comment by gummby8** [comment]
@idragonfire

Exactly right

**Comment by zupalex** [comment]
Someone explain on the plugin thread why the suggestion 1 is not possible. I quote :

Digi said: ↑
That was always the case because the game places the skin on entities with names that have skins, but to change the color of the overhead text, the plugin adds a character before the name so the skin disappears becase there isn't a plaer with that specific name.
So you can not separate them without client + server mods, like spout !

So looks like we won't see this kind of feature :(

**Comment by gummby8** [comment]
But I can /nick myself and it does not change my skin at all NPCs are pseudo players so I don't see why not.

**Comment by zupalex** [comment]
Do you have a mod client side installed ?
When you do that, do other players see the change ?

**Comment by Loserman778** [comment]
zupalex, its called using essentials. The essentials plugin has the /nick command

**Comment by zupalex** [comment]
Oh I see, i did not know. Thanks for letting me know. 
So with the essential plugin, you can /nick yourself and the change apply for everyone in the server i.e. you're not the only one that can see the change ?
If that's the case it's a server side change and the citizen plugin could just use that right ?
